backend/service-user/src/main/java/g55/cs3219/backend/userService/controller/AuthenticationController.java

```diff
@@ -26,7 +26,7 @@
 import java.util.stream.Collectors;
 
 // https://youtu.be/uZGuwX3St_c?si=hQ2vppx_ACMhrS7u
-@RequestMapping("/auth")
+@RequestMapping("/api/auth")
```

Comment:
Maybe we could put `/api` as the base path in the configuration instead of every controller? Not sure if that is possible in Spring Boot